### PR TITLE
Remove gender reference in tagline

### PR DESCRIPTION
**Pull request by christopher-dG**
I do think it's nice to have something indicating that we're not a women-only group, but I think we need something other than just "female and male". Open to suggestions!

### CODE REVIEW

**Comment by christopher-dG** [comment]
Okay so the two thoughts so far are:

1. Add a sentence to the tagline along the lines of "all are welcome to join" (current implementation)
2. Keep it as just the first sentence, but build on the inclusiveness point on the about page

Pros/cons

1:
- People don't need to go anywhere but the homepage to get the point
- It's a bit wordy maybe, might not look great on narrow phones (haven't tested)

2:
- Keeps the front page concise
- We run the risk of people misinterpreting and leaving without looking at the about page

**Comment by christopher-dG** [comment]
The diff got a bit ugly, I changed some indentation. There's just the one new sentence on the about-us page.

File: index.html [hunk 1]
```diff
@@ -12,21 +11,15 @@
     </div>
 </div>
 
-
-
-
 <div class="img-container">
     <img style="width:100%;" src="{{ site.baseurl }}/assets/img/home/so-helpful-cropped-all3.jpg"/>
-    <div class="bottom-left"><h1>We are a group of University of Manitoba students who support women in technology.</h1></div>
+    <div class="bottom-left"><h1>We are a group of University of Manitoba students who support women in technology. All are welcome to join!</h1></div>
```
Comment (stoykoj) [inline index.html]: I inspected the site and I'm not sure why this div is here... Unless someone else can find a purpose, can you just delete it?
